### PR TITLE
Move to the Revault organisation

### PR DESCRIPTION
**Pull request by darosior**
Based on #99 this prepares the move to github.com/revault

### CODE REVIEW

File: TODO [hunk 1]
```diff
@@ -0,0 +1,7 @@
+- Gloria's PR
+
+- Managers need to poll the coordinator for sigs too
+- Startup check to ping Coordinator
+- Signer for the functional tests
+- `getdescriptors`
+- Message size limitations
```
Comment (edouardparis) [inline TODO:7]: :)

Comment (darosior) [reply TODO:7]: d'oh

File: README.md [hunk 2]
```diff
@@ -2,24 +2,45 @@
...
+$ re stop
+```
+You can find more RPC commands at [`doc/API.md`](doc/API.md) but all aren't implemented
+yet!
```
Comment (edouardparis) [inline README.md:36]: I am not sure to like it in the README, can we create a wiki section or something else ?

Comment (darosior) [reply README.md:36]: This is temporary. For now there is nothing really warranting a wiki imo: it's just for lurkers to quickly try it out. Eventually i'd like to move all our doc in the doc/ folder and keep the README clean

**Comment by darosior** [comment]
Fixed a typo in CONTRIBUTING (we were linking to revault_tx, not revaultd)

**Review by edouardparis — APPROVED** [review]
ACK 888523f434e671587fb05713aec4cf018123b4b3

**Pull request merged**